internal/pkg/source/source.go

```diff
@@ -134,6 +134,10 @@ func sourcesFromBlocks(fn *ssa.Function, conf Classifier, taggedFields fieldtags
 				if isProducedBySanitizer(v.(ssa.Value), conf) {
 					continue
 				}
+			case *ssa.TypeAssert:
```

Comment:
Please add an empty line above to separate this case from the previous one, as well as a comment explaining why we need this specific case.

Reply:
Done.